backend/handleError.ts

```diff
@@ -35,7 +36,14 @@ export function handleError(res: NextApiResponse, e: unknown) {
     console.error(e);
     return res.status(400).json(e.issues);
   }
+  if (e instanceof AxiosError) {
+    console.error(e.response?.data);
+    return res.status(500).send('Axios Server error!');
+  }
+
+  if (e instanceof Error) {
+    return res.status(400).send(e.message);
+  }
```

Comment:
Lisäsin `AxiosError`in, koska yksi tapa käyttää Googlen API:a oli lähettää POST-request API-endpointtiin. Omasta mielestä kuitenkin tuo Googlen `googleapis`-kirjasto oli paljon näppärämpi.

Joten `AxiosError`:lle ei nyt ole käyttöä joten poistoon?



`Error`in lisäämisen lisäsin sen takia, koska serveri palautti vain "Server error!", jos tuli "tuntematon virhe"

Tuo itseasiassa ei liene kovinkaan järkevää, koska serverivirheissähän voi olla jotain ns. salattua tietoa. Joten kannattanee poistaa kanssa?

Reply:
Parempi käyttää `googleapis` kirjastoa kuin Axiota suoraan. Jos teet Axios-kutsun Google API:in ja se jostain syystä feilaa, niin sitä virheilmoitusta ei tosiaan ole syytä lähettää fronttiin.